version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "1.1.2-SNAPSHOT"
+ThisBuild / version := "1.1.2"
```

Comment:
Please revert this part. We're still trying to sort out publishing rights in #153